app/components/enter-hours.tsx

```diff
@@ -0,0 +1,298 @@
+                                                        <CalendarDays className="ml-auto h-4 w-4 opacity-50" />
+                                                    </Button>
+                                                </FormControl>
+                                            </PopoverTrigger>
+                                            <PopoverContent className="w-auto p-0" align="start">
+                                                <Calendar
+                                                    mode="single"
+                                                    selected={field.value}
+                                                    onSelect={field.onChange}
+                                                    disabled={(date) =>
+                                                        date > new Date() || date < new Date("1900-01-01")
+                                                    }
+                                                    initialFocus
+                                                />
+                                            </PopoverContent>
+                                        </Popover>
+                                        <FormMessage />
+                                    </FormItem>
+                                )}
+                            />
+                            <FormField
+                                control={form.control}
+                                name="startTime"
+                                render={({ field }) => (
+                                    <FormItem>
+                                        <FormLabel>Start Time</FormLabel>
+                                        <FormControl>
+                                            <Select
+                                                onValueChange={field.onChange}
+                                                defaultValue={field.value}
+                                                {...field}
+                                            >
+                                                <FormControl>
+                                                    <SelectTrigger>
+                                                        <SelectValue
+                                                            placeholder="Select a start time when you began work"
+                                                            className="grid"
+                                                        />
+                                                    </SelectTrigger>
+                                                </FormControl>
+                                                <SelectContent>
+                                                    <ScrollArea className="h-[300px]">
+                                                        <SelectItem value="12:00 AM">12:00 AM</SelectItem>
+                                                        <SelectItem value="12:30 AM">12:30 AM</SelectItem>
```

Comment:
You can follow the pattern here on avoiding having all these typed manually:

https://github.com/troypoulter/toil-calculator/blob/20bcab4be25d8e2a2c160e11441ec566f25cc62d/app/components/rulesets.tsx#L229-L233